packages/@aws-cdk-testing/framework-integ/test/aws-ecs-patterns/test/ec2/integ.tls-network-load-balanced-ecs-service.ts

```diff
@@ -0,0 +1,40 @@
+import { InstanceType, Vpc } from 'aws-cdk-lib/aws-ec2';
+import { Cluster, ContainerImage } from 'aws-cdk-lib/aws-ecs';
+import { App, Stack } from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { NetworkLoadBalancedEc2Service } from 'aws-cdk-lib/aws-ecs-patterns';
+import { Certificate } from 'aws-cdk-lib/aws-certificatemanager';
+
+/**
+ * In order to test this you need prepare a certificate.
+ */
+const certArn = process.env.CDK_INTEG_CERT_ARN || process.env.CERT_ARN;
+if (!certArn) throw new Error('For this test you must provide your own Certificate as an env var "CERT_ARN". See framework-integ/README.md for details.');
```

Comment:
I do not see the environment variable `CERT_ARN` mentioned in the framework-integ/README.md file. Can you update this file, and add some details on how to create this certificate.

Reply:
Thanks for the review!
I updated the README and attached a link to AWS doc.  

If we need more details in README, I propose to do it in another PR.  
So the ECS Fargate pattern fix can be published first.